MAINTAINERS.yml

```diff
@@ -1841,6 +1838,489 @@ West:
     labels:
         - "area: West"
 
+"West project: canopennode":
+  status: maintained
+  maintainers:
+    - henrikbrixandersen
+  files:
+    - modules/canopennode/
+  labels:
+    - manifest-canopennode
+    - "area: CAN"
+
+"West project: civetweb":
+  status: obsolete
+  files:
+    - samples/net/civetweb/
+    - modules/Kconfig.civetweb
+  labels:
+    - manifest-civetweb
+    - "area: Networking"
+    - "area: civetweb"
+
+"West project: cmsis":
+  status: odd fixes
+  collaborators:
+    - nashif
+  files:
+    - modules/Kconfig.cmsis
+    - modules/Kconfig.cmsis_dsp
+    - modules/Kconfig.cmsis_nn
+  labels:
+    - manifest-cmsis
+
+"West project: edtt":
+  status: odd fixes
+  collaborators:
+    - erbr-ot
+    - kruithofa
+    - thoh-ot
+    - MariuszSkamra
+  files: []
+  labels:
+    - manifest-edtt
+
+"West project: fatfs":
+  status: maintained
+  maintainers:
+    - de-nordic
+  files: []
+  labels:
+    - manifest-fatfs
+
+"West project: hal_altera":
+  status: obsolete
+  files:
+    - modules/Kconfig.altera
+  labels:
+    - manifest-hal_altera
+
+"West project: hal_atmel":
+  status: maintained
+  maintainers:
+    - nandojve
+  files:
+    - modules/Kconfig.atmel
+  labels:
+    - manifest-hal_atmel
+
+"West project: hal_cypress":
+  status: odd fixes
```

Comment:
pinging @npal-cy, should you be the maintainer?

Reply:
it should be @ifyall for both: hal_cypress/hal_infineon